### DISCUSSION TITLE
Get colored background based on cover art

### DISCUSSION THREAD

**Discussion by Hooooooman** [pear-devs/pear-desktop#535] [Ideas]
I think there should be a plugin that detects the album cover and colors the background similar to it
Example:
![145275173-3165d652-7bd4-444c-83ed-f9330f755229](https://user-images.githubusercontent.com/91048741/147881204-ccd9b3a7-c2f3-4df4-9a72-622bd1c8f4bf.png)
.

**Comment by inson1** [comment]
@Hooooooman done (partly, but that is another thing) may you close it? :)
